bare_metal_tests/TESTS/bringup/uart/test.py

```diff
@@ -0,0 +1,330 @@
+"""
+mbed SDK
+Copyright (c) 2017 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import time
+import serial
+import sys
+import mbed_lstools
+from subprocess import Popen, PIPE, STDOUT
+import shutil
+import argparse
+import ntpath
```

Comment:
Do not do this. It's not cross platform AT ALL. Use `os.path` instead.

Reply:
Removed redundant imports (including ntpath).